index.html

```diff
@@ -165,8 +165,32 @@ <h4 class="connect">
             <li>Frameworks<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
             <img src="images/Separator Bottom.svg" alt="" srcset="">
             <li>Skills<span class="angle"><img src="images/angle-right.svg" alt=""></span> </li>
-            <img src="images/Separator Bottom.svg" alt="" srcset="">
+            <img class="last-line" src="images/Separator Bottom.svg" alt="" srcset="">
         </ul>
     </section>
+    <section class="contact-section">
+        <img  class="c-eli1" src="images/contact-Ellipse1.svg" alt="">
+        <img class="c-eli2" src="images/contact-Rectangle1.svg" alt="">
+        <img class="c-eli3" src="images/contact-Ellipse2.svg" alt="">
+        <h3 class="contact-title"> Contact me</h3>
+        <div >
+            <p class="contact-text">If you have an application you are interested in developing, a feature that you need  built or a project that needs coding. 
+                I’d love to help with it!
+            </p> 
+        </div>
+        <form action="https://formspree.io/f/xbjezrzn" method="POST">
+            <div class="fields">
+                <li><input name="name" type="text" maxLength="30" placeholder="John Doe"></li>
+                <li><input type="email" name="email" id="email" placeholder="loremipsum@gmail.com"></li>
+                <li><textarea maxLength="500" name="message" id="" cols="30" rows="10" placeholder="Write your message here..."></textarea></li>
```

Comment:
- [ ] Kindly consider removing these `li`s as suggested by the previous reviewer and also the `li` without `ul` doesn't have any semantic meaning. they are redundant so please consider removing them. 
- [OPTIONAL] Kindly consider adding `required` attribute in all these three inputs. adding this client-side validation is one of the requirements and is missed by the previous reviewer so kindly consider adding that.